products/memgraph/upcoming/how_to_guides/import-data.md

```diff
@@ -5,6 +5,10 @@ imported either using the [CSV import tool](#import-csv) or using saved
 [cypher queries](#import-cypher) with `mg_client` running in non-interactive
 mode.
 
+NOTE: `mg_client` is deprecated tool still coming within Memgraph package.
```

Comment:
---
**NOTE**

`mg_client` is deprecated tool still coming within Memgraph package.
[mgconsole](https://github.com/memgraph/mgconsole) will replace `mg_client` in
the future. If possible, please use `mgconsole` instead.

---

Reply:
Because of GitBook https://docs.gitbook.com/editing-content/markdown#hints-and-callouts?

Reply:
Oh, didn't know about hints.
I'm okay with anything as long the notes are more noticeable... I like my suggestion more because it uses pure markdown but hints are also nice.